framework/JobHandler.py

```diff
@@ -42,12 +43,15 @@
 import Runners
 import Models
 # for internal parallel
-import pp
-import ppserver
+## TODO: REMOVE WHEN RAY AVAILABLE FOR WINDOWOS
+_rayAvail = False if imutil.find_spec("ray") is None else True
```

Comment:
This doesn't need changing, but is there a reason to "protect" this variable against inquiry from outside modules?